src/encoded/static/components/typeutils.js

```diff
@@ -296,23 +296,32 @@ export function fileStatusList(session, sessionProperties) {
 
 // Display supersedes/superseded_by lists.
 export const Supersede = ({ context }) => {
-    // Make array of superseded_by accessions
-    let supersededBys = [];
-    if (context.superseded_by && context.superseded_by.length) {
-        supersededBys = context.superseded_by.map(supersededBy => globals.atIdToAccession(supersededBy));
-    }
-
     // Make array of supersedes accessions
     let supersedes = [];
     if (context.supersedes && context.supersedes.length) {
         supersedes = context.supersedes.map(supersede => globals.atIdToAccession(supersede));
     }
 
-    if (supersededBys.length > 0 || supersedes.length > 0) {
+    let supersededByOutput = null;
```

Comment:
This is fine and me just thinking and typing up philosophy that might be wrong. I believe "undefined" means the variable has not been initialized and has never been used and null means it is initialized to nothing by some operation. The variable seems to fall under the former rather than the later. However, I have always seen people set the initial value to null

Reply:
I think it’s closer to the former because `supersededByOutput` gets assigned a React function if needed, and our standard has been to output null when there’s nothing to display. The React team does consider both null and undefined for outputting nothing https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored